addons/website_livechat/static/src/embed/livechat_button_patch.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="im_livechat.LivechatButton" t-inherit-mode="extension">
+        <xpath expr="//button[hasclass('o-livechat-LivechatButton')]" position="attributes">
+            <attribute name="class" remove="rounded-top-2 pt-3 pb-2 px-3" add="d-flex justify-content-center align-items-center shadow rounded-circle mb-5 fa fa-commenting" separator=" "/>
+            <attribute name="t-attf-style" add="font-size: 24px; width: 56px; height: 56px; min-width: 56px;"  separator=" "/>
+        </xpath>
+    </t>
```

Comment:
I'd need to check in more details, but just to make sure: aren't we mixing up having `website_livechat` merely installed, and the difference between being on the actual website or being in embed elsewhere?

Reply:
Side quest: can we remove `website_livechat` overrides as much as possible, trying to move to standard livechat all possible features?

Reply:
I don't think so. Before this refactoring, having the `website_livechat` installed would have updated the livechat button skin. As for the override, I'm up for removing it and making this design the standard one but since it requires to update the livechat backend (i.e. button text option) I'll add it to [this task](https://www.odoo.com/web#id=3332872&cids=1&menu_id=4720&action=333&active_id=1519&model=project.task&view_type=form).